cypress/features/regression/designSystem/toast.feature

```diff
@@ -0,0 +1,71 @@
+Feature: Toast Design System component
```

Comment:
If `component` will be tested using `knobs` - we should put it into `Test` directory